mods/content/xenobiology/slime/_slime.dm

```diff
@@ -292,6 +292,16 @@
 /mob/living/slime/check_has_mouth()
 	return 0
 
+/mob/living/slime/set_nutrition(amt)
+	..()
+	if(get_hydration() < amt)
+		set_hydration(amt)
+
+/mob/living/slime/set_hydration(amt)
```

Comment:
Why not just set the hydration rate to zero and opt out of hydration altogether? Doing `nutrition = max(nutrition, hydration); hydration = max(hydration, nutrition)` is a bit confusing and I'm not sure what it's meant to solve.

Reply:
Is this chaining, though? I feel like this could lead to weird edge cases given the conditional.